README.md

```diff
@@ -32,10 +32,15 @@ We have migrated to helm 3, make sure to install the correct package version (>=
 1. `aws configure --profile embl-ebi`
 	- See [Quickly Configure ASW CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html#cli-quick-configuration) for `AWS Access Key ID` & `AWS Secret Access Key`
 	- Set region to `us-east-1`
-1. See [this video](https://org-humancellatlas-sam.s3.amazonaws.com/HCA-prod-acct-setup-720p.mov) for configuring your connection to the DCP Developer Role.
- - Account: `ebi-web-dev`
- - Role: `ingest-devops`
- - ARN: `arn:aws:iam::871979166454:role/ingest-devops`
+2. Edit your `./aws/config` to look like this:
+```
+[profile embl-ebi]
+role_arn = arn:aws:iam::871979166454:role/ingest-devops
+source_profile = ebi
+
+[profile ebi]
+region = us-east-1
+```
 
```

Comment:
We can actually use a more appropriate profile name instead of `embl-ebi`. We just need to update the config files. (maybe later) 

The region can be set under `embl-ebi` profile. And could you also note the source profile contains the credentials of a user who can assume the ingest-devops role? Thanks!

Reply:
The region can be set in embl-ebi but for some reason you still need the ebi profile else this error occurs: `Partial credentials found in assume-role, missing: source_profile or credential_source`